app/src/main/java/com/arygm/quickfix/ui/uiMode/appContentUI/userModeUI/search/Announcement.kt

```diff
@@ -19,6 +19,7 @@ import androidx.compose.material.icons.filled.PhotoLibrary
 import androidx.compose.material3.*
```

Comment:
There the star imports should be removed for more precise imports (you can just run then optimise imports tool)